lib/bloc/transactions/value_transfer/vtt_create/vtt_create_bloc.dart

```diff
@@ -1,19 +1,19 @@
-import 'dart:isolate';
 
 import 'package:bloc/bloc.dart';
 import 'package:equatable/equatable.dart';
+import 'package:fixnum/fixnum.dart';
```

Comment:
This path is not correct and fixnum is not used, should it be deleted?

Reply:
fixnum is used for the Int64 type which is required in the protobuf schema https://github.com/witnet/witnet.dart/pull/5
https://github.com/witnet/witnet.dart/blob/protobuf/lib/src/schema/value_transfer_output.dart#L7-L8